assets/js/components/surveys/SurveyQuestionRatingChoice.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * SurveyQuestionRatingChoice component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../components/Button';
+
+const SurveyQuestionRatingChoice = ( { Icon, choice, answerQuestion } ) => {
+	const handleButtonClick = () => {
+		if ( typeof answerQuestion === 'function' ) {
+			answerQuestion( choice );
+		}
+	};
```

Comment:
Please wrap this handler with `useCallback`.

Reply:
👍